ui/app/pages/home/home.container.js

```diff
@@ -3,7 +3,7 @@ import { compose } from 'recompose'
 import { connect } from 'react-redux'
 import { withRouter } from 'react-router-dom'
 import { unconfirmedTransactionsCountSelector } from '../../selectors/confirm-transaction'
-``
```

Comment:
? not sure why that's there, but best to not patch if not needed.  Upstream is ok for things like that.